.github/workflows/build_hub.yml

```diff
@@ -4,11 +4,9 @@ on:
   pull_request:
     branches:
       - master
-    paths:
-      - "prime-router/**"
```

Comment:
deleting these will make the backend build on all frontend PR's

Reply:
@snesm Backend builds will be skipped because frontend changes won't flip `has_router_change` to true:

```    if: ${{ needs.pre_job.outputs.has_router_change == 'true' }}```

Reply:
Correct @JosiahSiegel . It turned out the conditionals had been in place already before I started working on this. Pretty much all I had to do is delete the dummy workflow.